### DISCUSSION TITLE
Getting Started to "Train a New Model for Accuracy"

### DISCUSSION THREAD

**Discussion by learnevolve** [explosion/spaCy#12774] [Help: Model Advice]
Hi Experts,

I am looking forward to Creating an English Model for Extracting Job info for the below Properites from the Job Descritions

1. Wages
2. Education
3. Experience
4. Mode (Remote/Hybrid/Onsite)
5. Job Type (fulltime/partime)
6. Shift (1st,2nd,3rd)

I have the job descriptions from the Indeed and Dice annotated with prodigy. 

I want to know the best pipeline to create the model can anyone suggest?

**Comment by danieldk** [comment]
Thank you for your question! Since this looks like a span classification task, the [spancat](https://spacy.io/api/spancategorizer) pipe is a good starting point. We also have a blog post on spancat, which describes in detail how spancat works:

https://explosion.ai/blog/spancat

If the set of patterns is relatively small and well-defined or if you don't have a lot of training data, you could also look at [SpanRuler](https://spacy.io/usage/rule-based-matching#spanruler), which allows you to write patterns that match spaCy token attributes.
